reworked thoughts, added seasonal thoughts, and added personality thoughts files affected kit general thoughts,newborn thoughts, warrior app thoughts, and general thoughts

Yzzuwyzzuf opened the pull request:
## About The Pull Request


edited almost all general thoughts categories

added kit to gen segment and removed the last 3 gen thoughts to move to a gen_not_kit as kits are perfectly capable of most things here

added segment for one cat to another for people who end up only having one cat in their clan whether due to a challenge or just unlucky leaving gen with only one cat thoughts

added gen not kit for thoughts kits shouldn't have given they've never left camp or worked a day in their little lives

added working cats thoughts for warriors, apps, deputies, and leaders who should have them but not kits, medicine cats, mediators, and elders who don't patrol the way these cats do or go to the gatherings

removed gen_to_alive and replaced it with gen_to_other

added gen thoughts for all personalities

added mate thoughts 

added mate forbidden thoughts for cats with med cat mates 

added sibling thoughts category

added sibling adult thoughts category 

added gen to alive med cat 2 for specifically warriors, deputies, and leaders for recounting the gathering to cats who should be updated on it. happens in the books as well

added gen to alive med cat apprentice

reworked most of the thoughts in gen to alive deputy and moved those to working_cat

brought gen_otherclan from 2 to 9 thoughts so thought that was not able and very cool

added positive thoughts to refugee to alive leader because I truly believe a refugee would appreciate the leader for taking them in rather then critisizing them

added dead sibling thoughts

added comforting a clan mate during a rough storm thought

added the seasonal thoughts I put in my first pr

-----
changed every instance of everycat to everyone for consistency purposes as both are used through the doc

Added kit dead kit 2 for kits who knew the kit who died+7 thoughts

Added kit med parent for kits whose parents are medicine cats +8 thoughts

Added guarding kit for kits who are bold enough to try to guard the camp or guard the fresh kill pile from cats who already ate+2 thoughts

Added 14 thoughts to gen kit

Added kit to kit segment for playing with others or tattling on another kit +5 thoughts

Kit older cats+3 thoughts

Kit alive med cat reworded the second thought +4 thoughts, "knocks over rc" moved to bullying_kit_other, Added "is in time out" twice on purpose

Added bullying kit other specifically for them bullying another kit +3 thoughts

Added Daring kit segment +18 thoughts

Daydreamer kit +1 thought

Impulsive kit +7 thoughts

Skittish kit +6 thoughts

Quiet thoughts +4 thoughts

Sweet kit +8

Unruly kit  +2 thoughts

Shy kit +1 thought

Attention seeker kit +2 thoughts 

Noisy kit +2 thoughts

Self conscious kit +4 thoughts

Polite kit +1 thought

Added tattletale kit segment specifically for kits trying to tell on cats for something they in particular were told they can't do despite the recipient being older then them

------

Gen warrior app+13 thoughts

Warrior apprentice to alive warrior +1 thought

Warrior apprentice to alive mentor +1 thought

----------

Gen newborn +5 thoughts

Gen newborn to parent +3

Added gen newborn to sibling +3 thought


## Why This Is Good For ClanGen

adds a ton of personality and makes some of the more confusing thought possibilities like kits seeing a big dog despite being in camp and very protected not possible

## Proof of Testing

my testing is working on my mod on the discord which has had a very great reception which inspired me to submit prs in the first place
![thot9](https://github.com/user-attachments/assets/ede3c3fb-9ffb-45ab-bd30-2e3d3ef7fa76)
![thot8](https://github.com/user-attachments/assets/7ef3ffea-f9b1-493b-ac4d-ff94b27b2a57)
![thot7](https://github.com/user-attachments/assets/acb9a3f0-a618-4136-808c-84ece5cc6b00)
![thot6](https://github.com/user-attachments/assets/19b841be-d05b-4de9-9823-8f8d2e21e8ee)
![thot5](https://github.com/user-attachments/assets/8639505b-49d7-4ccf-9c39-09cfcd875bc2)
![thot4](https://github.com/user-attachments/assets/8b4b17af-5c6f-4c6e-87ac-ca89b3c60465)
![thot3](https://github.com/user-attachments/assets/9d89e223-3d01-4102-b396-241edf7c0e96)
![thot2](https://github.com/user-attachments/assets/c2452d34-28a9-4cf6-8e5c-a3bfb0394c67)
![thot1](https://github.com/user-attachments/assets/1565eb4d-aef5-467f-96ff-b93cc8dea487)
![kit3](https://github.com/user-attachments/assets/e9d4bb8c-3259-4661-b0f9-2513a43637a1)
![kit2](https://github.com/user-attachments/assets/1231365e-39e0-413e-9475-820dd9e1807b)
![kit1](https://github.com/user-attachments/assets/9184ca8c-b632-46e1-b7b7-24264f99ccbb)


## Changelog/Credits

reorganized thoughts
added personality thoughts
added relationship thoughts

Yzzuwyzzuf committed:
Add files via upload

Yzzuwyzzuf committed:
Update names.json

added eye suffixes, color suffixes, pelt prefixes
reorganized names
added lots of loner names and warrior names in all categories

Yzzuwyzzuf committed:
Update names.json

forgot 2 quotation marks

Yzzuwyzzuf committed:
Update names.json

fixing forgotten commas

Yzzuwyzzuf committed:
Update names.json

reverted back to main file names doc

Yzzuwyzzuf committed:
Update and rename seasonalthoughts.txt to x

dunno how to delete this, leaving it at this for now as the contents are being added to other files

Yzzuwyzzuf committed:
Merge branch 'ClanGenOfficial:development' into Fuzzys-expanded-text-mod

Yzzuwyzzuf committed:
Update names.json

naming categories added, prefixes and suffixes rearranged, loner names added

Yzzuwyzzuf committed:
Update names.json

forgot to re-add the clan names thing originally in the doc but accidentally removed

Yzzuwyzzuf committed:
Update names.json

removed unnecessary commas

Yzzuwyzzuf committed:
Update general.json

added kit to gen segment and removed the last 3 gen thoughts to move to a gen_not_kit as kits are perfectly capable of most things here

added segment for one cat to another for people who end up only having one cat in their clan whether due to a challenge or just unlucky leaving gen with only one cat thoughts

added gen not kit for thoughts kits shouldn't have given they've never left camp or worked a day in their little lives

added working cats thoughts for warriors, apps, deputies, and leaders who should have them but not kits, medicine cats, mediators, and elders who don't patrol the way these cats do or go to the gatherings

removed gen_to_alive and replaced it with gen_to_other

added gen thoughts for all personalities

added mate thoughts 

added mate forbidden thoughts for cats with med cat mates 

added sibling thoughts category

added sibling adult thoughts category 

added gen to alive med cat 2 for specifically warriors, deputies, and leaders for recounting the gathering to cats who should be updated on it. happens in the books as well

added gen to alive med cat apprentice

reworked most of the thoughts in gen to alive deputy and moved those to working_cat

brought gen_otherclan from 2 to 9 thoughts so thought that was not able and very cool

added positive thoughts to refugee to alive leader because I truly believe a refugee would appreciate the leader for taking them in rather then critisizing them

added dead sibling thoughts

added comforting a clan mate during a rough storm thought

added the seasonal thoughts I put in my first pr

Yzzuwyzzuf committed:
Update kitten.json

Added kit dead kit 2 segment for kits who knew the kit who died+7 thoughts

Added kit med parent for kits whose parents are medicine cats +8 thoughts

Added guarding kit for kits who are bold enough to try to guard the camp or guard the fresh kill pile from cats who already ate+2 thoughts

Added 14 thoughts to gen kit

Added kit to kit segment for playing with others or tattling on another kit +5 thoughts

Kit older cats+3 thoughts

Kit alive med cat reworded the second thought +4 thoughts, "knocks over rc" moved to bullying_kit_other, Added "is in time out" twice on purpose

Added bullying kit other specifically for them bullying another kit +3 thoughts

Added Daring kit segment +18 thoughts

Daydreamer kit +1 thought

Impulsive kit +7 thoughts

Skittish kit changed "is hiding from everycat" and "hopes there's enough fresh-kill for everycat" to "is hiding from everycat" and "hopes there's enough fresh-kill for everyone" for consistency purposes +6 thoughts

Quiet thoughts +4 thoughts

Sweet kit +8

Unruly kit  +2 thoughts

Shy kit +1 thought

Attention seeker kit +2 thoughts 

Noisy kit +2 thoughts

Self conscious kit +4 thoughts

Polite kit +1 thought

Added tattletale kit segment specifically for kits trying to tell on cats for something they in particular were told they can't do despite the recipient being older then them

Yzzuwyzzuf committed:
Update apprentice.json

added a few thoughts

Yzzuwyzzuf committed:
Update newborn.json

Gen newborn +5 thoughts

Gen newborn to parent +3

Added gen newborn to sibling +3 thought

Yzzuwyzzuf committed:
Update names.json

slight fix, added one prefix (Tortoise)

Yzzuwyzzuf committed:
Delete x

deleted file I added by accident

Yzzuwyzzuf committed:
Update names.json

reverted to original names doc as names were not a part of this pr

keplerstelescope commented on resources/dicts/thoughts/alive/kitten.json:
```diff
@@ -30,6 +48,40 @@
+        "id": "guarding_kit",
+        "thoughts": [
+            "Has taken up the role of guarding the fresh-kill pile from cats wanting an extra piece and is taking {PRONOUN/m_c/poss} job very seriously",
+            "Is trying to force {PRONOUN/m_c/poss} way into guarding camp"
```
```suggestion
            "Is trying to force {PRONOUN/m_c/poss} way into being allowed to guard camp"
```

keplerstelescope commented on resources/dicts/thoughts/alive/kitten.json:
```diff
@@ -30,6 +48,40 @@
+        "thoughts": [
+            "Is visiting the medicine den to see {PRONOUN/m_c/poss} parent, r_c",
+            "Is being teased for having codebreakers as parents",
+            "Wonders if {PRONOUN/m_c/subject}'ll have prophecies like r_c",
```
```suggestion
            "Wonders if {PRONOUN/m_c/subject}'ll receive prophecies like r_c",
```

keplerstelescope commented on resources/dicts/thoughts/alive/kitten.json:
```diff
@@ -30,6 +48,40 @@
+            "Is visiting the medicine den to see {PRONOUN/m_c/poss} parent, r_c",
+            "Is being teased for having codebreakers as parents",
+            "Wonders if {PRONOUN/m_c/subject}'ll have prophecies like r_c",
+            "Wonders if StarClan hates {PRONOUN/m_c/object} for being a medicine cats' kit",
```
```suggestion
            "Wonders if StarClan hates {PRONOUN/m_c/object} for being a medicine cat's kit",
```

keplerstelescope commented on resources/dicts/thoughts/alive/kitten.json:
```diff
@@ -68,7 +120,40 @@
+            "kitten"
+        ],
+        "random_status_constraint":[
+            "Kitten"
```
```suggestion
            "kitten"
```

keplerstelescope commented on resources/dicts/thoughts/alive/kitten.json:
```diff
@@ -68,7 +120,40 @@
+            "Wants to join the hunting patrol but can't",
+            "Is learning how to use their claws",
+            "Tried climbing r_c",
+            "Is eating grass pretending to be prey",
```
```suggestion
            "Is eating grass, pretending to be prey",
```

keplerstelescope commented on resources/dicts/thoughts/alive/kitten.json:
```diff
@@ -96,7 +184,10 @@
             "Is asking the medicine cat why all the herbs have to be stinky and gross",
-            "Wants to follow the medicine cat everywhere"
+            "Wants to follow the medicine cat everywhere",
+            "Is asking r_c what every herb does but isnt retaining a word of it",
```
```suggestion
            "Is asking r_c what every herb does but isn't retaining a word of it",
```

keplerstelescope commented on resources/dicts/thoughts/alive/kitten.json:
```diff
@@ -145,7 +240,11 @@
             "Tells r_c that nobody likes {PRONOUN/r_c/object}",
-            "Is scratching everything {PRONOUN/m_c/subject} can get {PRONOUN/m_c/poss} claws on"
+            "Is scratching everything {PRONOUN/m_c/subject} can get {PRONOUN/m_c/poss} claws on",
+            "Is making r_c pretend to be prey for {PRONOUN/r_c/object} to hunt",
```
```suggestion
            "Is making r_c pretend to be prey for {PRONOUN/m_c/object} to hunt",
```

keplerstelescope commented on resources/dicts/thoughts/alive/kitten.json:
```diff
@@ -120,7 +211,11 @@
-            "Is begging r_c to play with {PRONOUN/m_c/object}"
+            "Is begging r_c to play with {PRONOUN/m_c/object}",
+            "Pounced on r_c",
+            "Runs out of energy for several seconds then gets back up energized",
```
```suggestion
            "Runs out of energy for several seconds... then hops back up, fully re-energized!",
```

keplerstelescope commented on resources/dicts/thoughts/alive/kitten.json:
```diff
@@ -120,7 +211,11 @@
+            "Is begging r_c to play with {PRONOUN/m_c/object}",
+            "Pounced on r_c",
+            "Runs out of energy for several seconds then gets back up energized",
+            "Nips at r_c then runs away giggling",
```
```suggestion
            "Nips at r_c, then runs away giggling",
```

keplerstelescope commented on resources/dicts/thoughts/alive/kitten.json:
```diff
@@ -154,6 +253,20 @@
+            "bullying"
+        ],
+        "random_status_constraint":[
+            "Kitten"
```
```suggestion
            "kitten"
```

keplerstelescope commented on resources/dicts/thoughts/alive/kitten.json:
```diff
@@ -177,6 +290,34 @@
         ]
     },
+    {
+        "id": "daring_kit",
```
can kits have the daring trait? if not i think this should be merged into fearless_kit and/or maybe impulsive_kit?

Yzzuwyzzuf replied:
the wiki says daring is a possible kit trait. I wont remove until its confirmed it isn't in dev version but I will add the thoughts to the other traits too or vice versa

keplerstelescope commented on resources/dicts/thoughts/alive/kitten.json:
```diff
@@ -177,6 +290,34 @@
+            "Tries to sneak out of camp with a patrol",
+            "Pounced on r_c",
+            "Is boldly walking out of camp",
+            "Is demandind others spar with {PRONOUN/m_c/object}",
```
```suggestion
            "Is demanding others spar with {PRONOUN/m_c/object}",
```

keplerstelescope commented on resources/dicts/thoughts/alive/kitten.json:
```diff
@@ -252,7 +393,14 @@
+            "Picked up something nodody else got a good look at and is now running from several cats",
+            "Is pretending to be a mouse and trying to get others to chase {PRONOUN/m_c/object}",
+            "Nips at r_c and runs away giggling",
+            "rearranges all of the herbs as a prank"
```
```suggestion
            "Rearranges all of the herbs as a prank"
```

keplerstelescope commented on resources/dicts/thoughts/alive/kitten.json:
```diff
@@ -252,7 +393,14 @@
+            "Is zooming around",
+            "Pounced on r_c",
+            "Is practicing {PRONOUN/m_c/poss} pounce on r_c's tail",
+            "Is complaining of a tummy ache after sneaking into the medicine cats herbs and eating one",
```
```suggestion
            "Is complaining of a tummyache after sneaking into the medicine den's herb storage and eating one",
```
can't really specify "medicine cat's" or "medicine cats'" since afaik there's no way to tell how many medicine cats there are from here

keplerstelescope commented on resources/dicts/thoughts/alive/kitten.json:
```diff
@@ -420,7 +589,9 @@
             "Begs for life-saving herbs to cure {PRONOUN/m_c/poss} minor scrape",
-            "Is calling a Clan meeting, before r_c rudely interrupts {PRONOUN/m_c/object}"
+            "Is calling a Clan meeting, before r_c rudely interrupts {PRONOUN/m_c/object}",
+            "Is demanding r_c's attention even though {PRONOUN/r_c/subject} {VERB/r_c/are/is} busy",
```
```suggestion
            "Is demanding r_c's attention even though {PRONOUN/r_c/subject}{VERB/r_c/'re/'s} busy",
```
minor nitpick, flows slightly better this way (feel free to ignore this suggestion if you want)

keplerstelescope commented on resources/dicts/thoughts/alive/kitten.json:
```diff
@@ -252,7 +393,14 @@
+            "Pounced on r_c",
+            "Is practicing {PRONOUN/m_c/poss} pounce on r_c's tail",
+            "Is complaining of a tummy ache after sneaking into the medicine cats herbs and eating one",
+            "Picked up something nodody else got a good look at and is now running from several cats",
```
```suggestion
            "Picked up something nobody else got a good look at and is now running from several cats",
```

keplerstelescope commented on resources/dicts/thoughts/alive/kitten.json:
```diff
@@ -400,7 +568,8 @@
             "Wants to stay in {PRONOUN/m_c/poss} nest today",
-            "Thought about talking to r_c but changed {PRONOUN/m_c/poss} mind"
+            "Thought about talking to r_c but changed {PRONOUN/m_c/poss} mind",
+            "Is watching cats go about their day"
```
```suggestion
            "Is watching cats go about their days"
```

Yzzuwyzzuf replied:
I think I'd still prefer day over days

keplerstelescope commented on resources/dicts/thoughts/alive/newborn.json:
```diff
@@ -6,7 +6,12 @@
+            "Head empty",
+            "Almost rolled out of the nest",
+            "Tried suckling on a piece of bedding",
+            "Squealing loudly",
```
```suggestion
            "Is squealing loudly",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -2,7 +2,8 @@
         "id": "gen",
         "thoughts": [
-            "Is feeling quite lazy",
+           "Is feeling quite lazy",
```
```suggestion
            "Is feeling quite lazy",
```
minor nitpick

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Is debating on which prey to take from the fresh-kill pile",
+            "Sits quietly watching cats go about their day",
+            "Is wondering if there are clans outside of the ones around {PRONOUN/m_c/poss} area",
+            "Is wondering if Starclan is watching cats outside the clans",
```
```suggestion
            "Is wondering if StarClan is watching cats outside the clans",
```

keplerstelescope replied:
```suggestion
            "Is wondering if StarClan is watching over cats outside the Clans",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Is watching the sky for rain",
+            "Is cloudwatching",
+            "Is debating on which prey to take from the fresh-kill pile",
+            "Sits quietly watching cats go about their day",
```
```suggestion
            "Sits quietly, watching cats go about their day",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Is cloudwatching",
+            "Is debating on which prey to take from the fresh-kill pile",
+            "Sits quietly watching cats go about their day",
+            "Is wondering if there are clans outside of the ones around {PRONOUN/m_c/poss} area",
```
```suggestion
            "Is wondering if there are Clans other than the ones around {PRONOUN/m_c/poss} area",
```
changing "outside of" to "other than" is a minor nitpick

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+        "thoughts": [
+            "Is asking r_c an important question",
+            "Can't find r_c anywhere",
+            "Is fetching r_c for another cat who was looking for {PRONOUN/m_c/object}",
```
```suggestion
            "Is fetching r_c for another cat who was looking for {PRONOUN/r_c/object}",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+        "id": "gen_not_kit",
+        "thoughts": [
+            "Is thinking on {PRONOUN/m_c/poss} life choices",
+            "Wishes {PRONOUN/m_c/subject} had the energy {PRONOUN/m_c/subject} had as a kit",
```
i don't know if this is a thought an adolescent (or potentially even a young adult) would have? maybe constrain it to certain grumpy/low-energy traits for adolescents/young adults?

Yzzuwyzzuf replied:
I mainly thought to have it there cause apprentices probably glamorized and underestimated how hard apprentice work would be ad it tired them out but I'll see about moving it to a more fitting category :)

Yzzuwyzzuf replied:
moved to a new category

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Wishes {PRONOUN/m_c/subject} had the energy {PRONOUN/m_c/subject} had as a kit",
+            "Found a wonderful napping spot outside camp",
+            "Had a nightmare involving a Twoleg trap",
+            "Is humming a tune {PRONOUN/m_c/subject} heard near a twolegplace",
```
```suggestion
            "Is humming a tune {PRONOUN/m_c/subject} heard near a Twoleg place",
```
other thoughts use "Twoleg place", not "Twolegplace"; i'm not certain which one is corrected/intended, but suggesting editing it here for consistency

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Had a nice chat with a kittypet",
+            "Is on a walk out in the territory",
+            "Is having a nice conversation with the cats guarding camp",
+            "Is sitting near the camp entrance keeping an eye out for sneaky kits",
```
```suggestion
            "Is sitting near the camp entrance, keeping an eye out for sneaky kits",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
             "Saw a big scary dog today",
-            "Is looking forward to the next Gathering"
+            "Is drinking water from a nearby stream",
+            "Thought {PRONOUN/m_c/subject} saw a cat but turns out it was a twoleg object that looked like one"
```
```suggestion
            "Thought {PRONOUN/m_c/subject} saw a cat, but turns out it was a Twoleg object that looked like one"
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Is helping out around camp in between {PRONOUN/r_c/poss} duties",
+            "Is wanting to train",
+            "Wants to go hunt",
+            "Dosen't want to go on dawn patrol",
```
```suggestion
            "Doesn't want to go on dawn patrol",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Is wanting to train",
+            "Wants to go hunt",
+            "Dosen't want to go on dawn patrol",
+            "Dosen't want to go on dusk patrol",
```
```suggestion
            "Doesn't want to go on dusk patrol",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Is exhausted after a patrol",
+            "Is wanting to go on a patrol",
+            "Is pretending to look busy...",
+            "Is helping out around camp in between {PRONOUN/r_c/poss} duties",
```
```suggestion
            "Is helping out around camp in between {PRONOUN/m_c/poss} duties",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Is daydreaming about a Clan celebration in {PRONOUN/m_c/poss} honor",
+            "Is asking the Clan leader what {PRONOUN/m_c/subject} can do to help out around camp",
+            "Has been imitating the Clan leader's behaviour recently",
+            "Knows they're a shoe in for deputy",
```
```suggestion
            "Knows {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} a shoo-in for deputy",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Is asking the Clan leader what {PRONOUN/m_c/subject} can do to help out around camp",
+            "Has been imitating the Clan leader's behaviour recently",
+            "Knows they're a shoe in for deputy",
+            "Offers to help clanmates with their duties",
```
```suggestion
            "Offers to help {PRONOUN/m_c/poss} Clanmates with their duties",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+        "id": "arrogant_gen",
+        "thoughts": [
+            "Is sure of {PRONOUN/m_c/poss} skills being better then anyone else",
+            "Is confidently spreading misinformaton on something {PRONOUN/m_c/subject} know little about",
```
```suggestion
            "Is confidently spreading misinformation on something {PRONOUN/m_c/subject} {VERB/m_c/know/knows} little about",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+        "thoughts": [
+            "Is sure of {PRONOUN/m_c/poss} skills being better then anyone else",
+            "Is confidently spreading misinformaton on something {PRONOUN/m_c/subject} know little about",
+            "Is confidently spreading misinformaton on something {PRONOUN/m_c/subject} know nothing about",
```
```suggestion
            "Is confidently spreading misinformation on something {PRONOUN/m_c/subject} {VERB/m_c/know/knows} nothing about",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Winked playfully at a Clanmate from across the clearing!",
+            "Has received several invitations to share tongues... and is deciding which to take",
+            "Is basking in a sunray in the camp clearing",
+            "Has the kits and apprentices very engaged in a very, very tall tale",
```
feels odd to have this as a thought an apprentice/adolescent could have?

Yzzuwyzzuf replied:
I was thinking an apprentice could be telling a story to other apprentices along the lines of almost catching a huge rabbit or taking down a bear all on their own. I definitely will change the wording/add a new category or two to help make things clearer though

Yzzuwyzzuf replied:
update, adding an older charismatic gen section and made it one on one instead of referring to multiple of both as chances are people might not have both

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -1182,7 +2588,11 @@
         "thoughts": [
-            "Remembers how r_c healed {PRONOUN/m_c/object}"
+            "Remembers how r_c healed {PRONOUN/m_c/object}",
+            "Hopes r_c is enjoying looking through StarClans herb storage",
```
```suggestion
            "Hopes r_c is enjoying looking through StarClan's herb storage",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -1796,6 +3232,152 @@
+        "id": "gen_greenleaf",
+        "thoughts": [
+            "Enjoying the warmth of Greenleaf",
+            "Is finding the buzzing of the cicadas annoying",
```
should this be biome-specific?

Yzzuwyzzuf replied:
Id say so, yes since cicadas mainly appear during the summer though that's just speaking from my experience

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -1796,6 +3232,152 @@
+            "Enjoying the warmth of Greenleaf",
+            "Is finding the buzzing of the cicadas annoying",
+            "Can't sleep because of the bugs and the heat",
+            "Purrs as a cool breeze its {PRONOUN/m_c/object}",
```
```suggestion
            "Purrs as a cool breeze hits {PRONOUN/m_c/object}",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -1922,13 +3505,18 @@
-            "Listened to bird songs all morning",
+            "Listened to birdsong all morning",
             "Just heard a frog croaking in camp!",
+            "Didn't know {PRONOUN/m_c/subject} were staring blankly at r_c",
```
```suggestion
            "Didn't know {PRONOUN/m_c/subject} {VERB/m_c/were/was} staring blankly at r_c",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -2210,16 +3801,22 @@
             "Is frustrated by {PRONOUN/m_c/poss} health",
             "Hisses at r_c for seemingly no reason",
-            "Has been experiencing changes in {PRONOUN/m_c/poss} behavior",
+            "Has been experiencing changes in {PRONOUN/m_c/poss} behaviour",
```
```suggestion
            "Has been experiencing changes in {PRONOUN/m_c/poss} behavior",
```
https://github.com/ClanGenOfficial/clangen/wiki/%5BWriting%5D-%E2%80%90-Basic grammar rules indicates american spelling of words

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -2210,16 +3801,22 @@
+            "Feels stronger today",
+            "Can't force {PRONOUN/m_c/self} out of {PRONOUN/m_c/poss} nest today",
+            "Is resigned to their fate",
+            "Prays to StarClan for just a little for more time...",
```
```suggestion
            "Prays to StarClan for just a little more time...",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -1594,7 +3021,8 @@
         "thoughts": [
-            "Is asking if {PRONOUN/m_c/subject} {VERB/m_c/need/needs} more training"
+            "Is asking if {PRONOUN/m_c/subject} {VERB/m_c/need/needs} more training",
+            "Never forgets to remind {PRONOUN/m_c/poss} clanmates of things they might have forgotten about"
```
```suggestion
            "Never forgets to remind {PRONOUN/m_c/poss} Clanmates of things they might have forgotten about"
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -1460,7 +2874,12 @@
             "Is licking {PRONOUN/m_c/poss} chest in embarrassment after being praised by {PRONOUN/m_c/poss} mentor",
-            "Is asking {PRONOUN/m_c/poss} mentor what {PRONOUN/m_c/subject} can do to help around camp today"
+            "Gently reminds a clanmate of something they forgot to do",
+            "Never forgets to thank StarClan for the food",
```
```suggestion
            "Never forgets to thank StarClan for {PRONOUN/m_c/poss} fresh-kill",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -1460,7 +2874,12 @@
         "thoughts": [
             "Is licking {PRONOUN/m_c/poss} chest in embarrassment after being praised by {PRONOUN/m_c/poss} mentor",
-            "Is asking {PRONOUN/m_c/poss} mentor what {PRONOUN/m_c/subject} can do to help around camp today"
+            "Gently reminds a clanmate of something they forgot to do",
```
```suggestion
            "Gently reminds a Clanmate of something they forgot to do",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -1460,7 +2874,12 @@
+            "Volunteers to watch the kits",
+            "Offers to help everyone {PRONOUN/m_c/subject} come across",
+            "Is asking {PRONOUN/m_c/poss} mentor what {PRONOUN/m_c/subject} can do to be helpful around camp today",
+            "Is the first to freshen the elders bedding"
```
```suggestion
            "Is the first to offer to freshen the elders bedding"
```
minor nitpick

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -805,15 +2197,24 @@
         "id": "gen_former_kittypet",
         "thoughts": [
             "Is reflecting on how different {PRONOUN/m_c/poss} life is now",
+            "Misses how {PRONOUN/m_c/poss} twolegs kept their den at a comfy temperature at all times",
```
```suggestion
            "Misses how {PRONOUN/m_c/poss} Twolegs kept their den at a comfy temperature at all times",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -805,15 +2197,24 @@
             "Is reflecting on how different {PRONOUN/m_c/poss} life is now",
+            "Misses how {PRONOUN/m_c/poss} twolegs kept their den at a comfy temperature at all times",
+            "Misses not having to work for {PRONOUN/m_c/poss} food each day",
+            "Wonders if {PRONOUN/m_c/poss} twolegs miss {PRONOUN/m_c/object}",
```
```suggestion
            "Wonders if {PRONOUN/m_c/poss} Twolegs miss {PRONOUN/m_c/object}",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -805,15 +2197,24 @@
             "Wishes others would stop thinking that {PRONOUN/m_c/subject}{verb/m_c/'re/'s} soft because of {PRONOUN/m_c/poss} kittypet past",
             "Wistfully thinks of {PRONOUN/m_c/poss} soft kittypet nest while trying to nap",
             "Wonders what became of {PRONOUN/m_c/poss} Twolegs",
+            "Is missing the pets {PRONOUN/m_c/poss} twolegs gave {PRONOUN/m_c/object}",
```
```suggestion
            "Is missing the pets {PRONOUN/m_c/poss} Twolegs gave {PRONOUN/m_c/object}",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -805,15 +2197,24 @@
             "Wistfully thinks of {PRONOUN/m_c/poss} soft kittypet nest while trying to nap",
             "Wonders what became of {PRONOUN/m_c/poss} Twolegs",
+            "Is missing the pets {PRONOUN/m_c/poss} twolegs gave {PRONOUN/m_c/object}",
+            "Is admittedly missing the toys {PRONOUN/m_c/poss} twolegs gave {PRONOUN/m_c/object}",
```
```suggestion
            "Is admittedly missing the toys {PRONOUN/m_c/poss} Twolegs gave {PRONOUN/m_c/object}",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -716,7 +2104,9 @@
-            "Is thankful for the safety of the camp"
+            "Is thankful for the safety of the camp",
+            "Woke up in a panic",
+            "Has been on edge watching the camp entrance all day"
```
```suggestion
            "Has been on edge, watching the camp entrance all day"
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -666,7 +2050,10 @@
         "thoughts": [
-            "Is wondering how the cats {PRONOUN/m_c/subject} helped in the past are doing"
+            "Is wondering how the cats {PRONOUN/m_c/subject} helped in the past are doing",
+            "Is helping out in the medicine cat den",
```
```suggestion
            "Is helping out in the medicine den",
```
minor nitpick

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -612,6 +1986,9 @@
             "Is grateful for time spent with r_c",
             "Is reminiscing about r_c",
+            "Smiles knowing r_c is reunited with {PRONOUN/r_c/poss} family now",
+            "Hopes r_c's joints feel better now that {PRONOUN/m_c/subject} joint StarClan's ranks",
```
```suggestion
            "Hopes r_c's joints feel better now that {PRONOUN/m_c/subject} {VERB/m_c/have/has} joined StarClan's ranks",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -527,6 +1899,7 @@
             "Is mourning the lost potential of r_c",
             "Is listening to stories about r_c",
             "Is learning more about r_c",
+            "Is mourning the fact r_c will never get {PRONOUN/r_c/object} warrior name",
```
```suggestion
            "Is mourning the fact r_c will never get {PRONOUN/r_c/poss} warrior name",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -298,13 +1608,15 @@
+            "Wishes r_c was {PRONOUN/m_c/poss} apprentice",
             "Doesn't want to go on patrol with r_c",
-            "Thinks r_c needs to work on {PRONOUN/r_c/poss} stalking technique",
+            "Thinks r_c needs to work on their stalking technique",
```
```suggestion
            "Thinks r_c needs to work on {PRONOUN/r_c/poss} stalking technique",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "took a nap with r_c",
+            "Took a nap in r_c's nest",
+            "Wants to work hard to impress r_c",
+            "Had a pointless petty argument with r_c",
```
```suggestion
            "Had a pointless, petty argument with r_c",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
-            "Asks r_c for pain relief",
-            "Wishes {PRONOUN/m_c/subject} understood what r_c is doing"
+            "Is sharing prey with r_c",
+            "took a nap with r_c",
```
```suggestion
            "Took a nap with r_c",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
-        ]
+            "Is urging r_c to rest",
+            "Brought r_c some prey",
+            "Wonders if {PRONOUN/m_c/subject}'ll still be accepted into Starclan after taking r_c as a mate",
```
```suggestion
            "Wonders if {PRONOUN/m_c/subject}'ll still be accepted into StarClan after taking r_c as a mate",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Brought r_c some prey",
+            "Wonders if {PRONOUN/m_c/subject}'ll still be accepted into Starclan after taking r_c as a mate",
+            "Hopes r_c will still be accepted into StarClan",
+            "Hopes starclan will forgive r_c and {PRONOUN/m_c/object} for becoming mates",
```
```suggestion
            "Hopes StarClan will forgive {PRONOUN/m_c/object} and r_c for becoming mates",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
     {
-        "id": "gen_alive_not_working_insteadinthemedden",
-        "not_working": true,
+        "id": "gen_mate_forbidden",
```
i don't think that medicine cats having mates is forbidden in ClanGen? (minor nitpick)

Yzzuwyzzuf replied:
I know but in the books its forbidden so I thought that wording fit. I can change it to codebreaker or something different if you'd prefer

Yzzuwyzzuf replied:
renamed gen_mate_medicine_cat

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Loves r_c so much",
+            "Is napping with r_c",
+            "Snuck away from camp with r_c for some alone time",
+            "Is nuzzling into r_c's fur taking in {PRONOUN/r_c/poss} scent",
```
```suggestion
            "Is nuzzling into r_c's fur, taking in {PRONOUN/r_c/poss} scent",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Brought r_c {PRONOUN/r_c/poss} favorite prey",
+            "Is thinking about having kits with r_c",
+            "Brought a flower back to camp for r_c",
+            "Is pressing against r_c purring loudly",
```
```suggestion
            "Is pressing against r_c, purring loudly",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Is causing problems",
+            "Recently put a dead snake at the camp entrance to scare Clanmates",
+            "Is embarrassed after getting a taste of {PRONOUN/m_c/poss} own bitter herbs... Serves {PRONOUN/m_c/object} right!",
+            "Taunted a dog stuck in a twoleg's fence",
```
```suggestion
            "Taunted a dog stuck in a Twoleg fence",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+        "id": "strict_warrior_to_living_apprentices",
+        "thoughts": [
+            "Snaps at some apprentices after they pull a harmless prank",
+            "Is critisizing everything r_c is doing wrong"
```
```suggestion
            "Is criticizing everything r_c is doing wrong"
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Is grooming {PRONOUN/m_c/self}, making sure every whisker is in place",
+            "Is busy chastising Clanmates... but no cat is sure what for",
+            "Can't stand to watch the younger cats make fools of themselves",
+            "Goes around camp policing everyone's posture",
```
```suggestion
            "Goes around camp, policing everyone's posture",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+        "id": "gen_sneaky",
+        "thoughts": [
+            "Took an extra piece of fresh-kill from the pile without anyone noticing",
+            "Taunted a dog stuck in a twoleg's fence",
```
```suggestion
            "Taunted a dog stuck in a Twoleg fence",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Is drawing symbols in the dirt with {PRONOUN/m_c/poss} claws",
+            "Is sitting quietly on the far side of camp",
+            "Paces back and forth",
+            "Wonders if everyone thinks their own thoughts or if starclan puts their thoughts in their heads",
```
```suggestion
            "Wonders if everyone thinks their own thoughts, or if StarClan puts their thoughts in their heads",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Is patiently listening to someone talk about things they are unquestionably wrong about and dying inside",
+            "Suggests a clanmate speak with a mediator",
+            "Was brutally honest with a clanmate and feels bad about it",
+            "Is spending a lot of {PRONOUN/m_c/poss} time thinking of ways to support any cat no matter what they're going through"
```
```suggestion
            "Is spending a lot of {PRONOUN/m_c/poss} time thinking of ways to support any cat, no matter what they're going through"
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+    {
+        "id": "sincere_gen",
+        "thoughts": [
+            "Is making rounds around camp checking in on every cat",
```
```suggestion
            "Is making rounds around camp, checking in on every cat",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Is lecturing about something someone said wrong offhandedly",
+            "Is bragging about how great a hunter {PRONOUN/m_c/subject} are",
+            "Announced that {PRONOUN/m_c/subject} {VERB/m_c/are/is} heading to the dirtplace",
+            "Taunted a dog stuck in a twoleg's fence",
```
```suggestion
            "Taunted a dog stuck in a Twoleg fence",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Hopes to help guide {PRONOUN/m_c/poss} Clanmates down the right path",
+            "Always cheers the loudest of any cat at naming ceremonies",
+            "Silently judges a clanmate from afar",
+            "Wonders if everyone thinks their own thoughts or if starclan puts their thoughts in their heads",
```
```suggestion
            "Wonders if everyone thinks their own thoughts, or if StarClan puts their thoughts in their heads",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Is reevaluating {PRONOUN/m_c/poss} morals",
+            "Hopes to help guide {PRONOUN/m_c/poss} Clanmates down the right path",
+            "Always cheers the loudest of any cat at naming ceremonies",
+            "Silently judges a clanmate from afar",
```
```suggestion
            "Silently judges a Clanmate from afar",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+        "thoughts": [
+            "Is finding ways to rebel against the instructions {PRONOUN/m_c/subject} were given",
+            "Is doubting why things are done the way they are",
+            "Taunted a dog stuck in a twoleg's fence",
```
```suggestion
            "Taunted a dog stuck in a Twoleg fence",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Is finding ways to rebel against the instructions {PRONOUN/m_c/subject} were given",
+            "Is doubting why things are done the way they are",
+            "Taunted a dog stuck in a twoleg's fence",
+            "Started an argument with the leader during a clan meeting",
```
```suggestion
            "Started an argument with the leader during a Clan meeting",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Is doubting why things are done the way they are",
+            "Taunted a dog stuck in a twoleg's fence",
+            "Started an argument with the leader during a clan meeting",
+            "Is spreading doubt through {PRONOUN/m_c/poss} clanmates over the way things are done",
```
```suggestion
            "Is spreading doubt through {PRONOUN/m_c/poss} Clanmates over the way things are done",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Is planning to wake up early tomorrow",
+            "Was the first to rise this morning",
+            "Reminds an apprentice to follow the code",
+            "Gently reminds a clanmate of something they forgot about",
```
```suggestion
            "Gently reminds a Clanmate of something they forgot about",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Was the first to rise this morning",
+            "Reminds an apprentice to follow the code",
+            "Gently reminds a clanmate of something they forgot about",
+            "Never forgets to thank StarClan for the food",
```
```suggestion
            "Never forgets to thank StarClan for {PRONOUN/m_c/poss} fresh-kill",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Isn't sure that {PRONOUN/m_c/subject} should be idling around at the moment",
+            "Is planning on going to sleep early tonight",
+            "Is making sure no work is being shirked!",
+            "Is offering to help clanmates with their duties",
```
```suggestion
            "Is offering to help Clanmates with their duties",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+        "id": "playful_warrior_to_living_kit",
+        "thoughts": [
+            "Plays mossball with r_c",
+            "Pretending to not see r_c sneaking up on {PRONOUN/m_c/object}"
```
```suggestion
            "Is pretending to not see r_c sneaking up on {PRONOUN/m_c/object}"
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Is cracking jokes",
+            "Taunted a dog stuck in a twoleg's fence",
+            "Is chasing {PRONOUN/m_c/poss} own tail, making others laugh",
+            "Bats at a clanmates tail",
```
```suggestion
            "Bats at a Clanmates tail",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Is chasing a bug around the camp clearing",
+            "Has been distracting {PRONOUN/m_c/poss} fellow Clanmates lately",
+            "Is cracking jokes",
+            "Taunted a dog stuck in a twoleg's fence",
```
```suggestion
            "Taunted a dog stuck in a Twoleg fence",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Is happily chasing a butterfly",
+            "Is running after a colorful beetle",
+            "Is letting out mrrows of laughter",
+            "Has been playing tricks on {PRONOUN/m_c/poss} fellow clanmates",
```
```suggestion
            "Has been playing tricks on {PRONOUN/m_c/poss} fellow Clanmates",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Is extra irritable today",
+            "Is extra huffy while working",
+            "Laughed when {PRONOUN/m_c/subject} saw a cat trip",
+            "Can't stand {PRONOUN/m_c/poss} clanmates today especially",
```
```suggestion
            "Can't stand {PRONOUN/m_c/poss} Clanmates today especially",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Laughed when {PRONOUN/m_c/subject} saw a cat trip",
+            "Can't stand {PRONOUN/m_c/poss} clanmates today especially",
+            "Is looking extra grumpy today",
+            "Is trying {PRONOUN/m_c/poss} hardest to avoid being bothered by {PRONOUN/m_c/poss} clanmates"
```
```suggestion
            "Is trying {PRONOUN/m_c/poss} hardest to avoid being bothered by {PRONOUN/m_c/poss} Clanmates"
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Wonders if everyone laughs with {PRONOUN/m_c/object} or at {PRONOUN/m_c/object}...",
+            "Is patiently listening to someone talk about things they are unquestionably wrong about and dying inside",
+            "Had a nightmare about being on {PRONOUN/m_c/poss} own",
+            "Had a nightmare about the entire clan looking to {PRONOUN/m_c/object} for what to do",
```
```suggestion
            "Had a nightmare about the entire Clan looking to {PRONOUN/m_c/object} for what to do",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Is feeling down...",
+            "Feels bad for staying in {PRONOUN/m_c/poss} nest a few minutes longer then usual",
+            "Wonders if everyone laughs with {PRONOUN/m_c/object} or at {PRONOUN/m_c/object}...",
+            "Is keeping to {PRONOUN/m_c/self} not wanting to ruin {PRONOUN/m_c/poss} clanmates moods with {PRONOUN/m_c/poss} gloominess",
```
```suggestion
            "Is keeping to {PRONOUN/m_c/self}, not wanting to ruin {PRONOUN/m_c/poss} Clanmates' moods with {PRONOUN/m_c/poss} gloominess",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+        "thoughts": [
+            "Is catching the attention of every cat around {PRONOUN/m_c/object}",
+            "Works to show off {PRONOUN/m_c/poss} skills in every movement {PRONOUN/m_c/subject} {VERB/m_c/make/makes}",
+            "Carefully curates {PRONOUN/m_c/poss} appearance to be the most dazzling in the clan",
```
```suggestion
            "Carefully curates {PRONOUN/m_c/poss} appearance to be the most dazzling in the Clan",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Is confidently spreading misinformaton on something {PRONOUN/m_c/subject} know little about",
+            "Is confidently spreading misinformaton on something {PRONOUN/m_c/subject} know nothing about",
+            "Is lecturing about something someone said wrong offhandedly",
+            "Carefully curates {PRONOUN/m_c/poss} appearance to be the most dazzling in the clan",
```
```suggestion
            "Carefully curates {PRONOUN/m_c/poss} appearance to be the most dazzling in the Clan",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Is catching the attention of every cat around {PRONOUN/m_c/object}",
+            "Works to show off {PRONOUN/m_c/poss} skills in every movement {PRONOUN/m_c/subject} {VERB/m_c/make/makes}",
+            "Carefully curates {PRONOUN/m_c/poss} appearance to be the most dazzling in the clan",
+            "Is playfully teasing {PRONOUN/m_c/poss} clanmates",
```
```suggestion
            "Is playfully teasing {PRONOUN/m_c/poss} Clanmates",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Carefully curates {PRONOUN/m_c/poss} appearance to be the most dazzling in the clan",
+            "Is playfully teasing {PRONOUN/m_c/poss} clanmates",
+            "Told a hilarious joke that spread around camp like a wildfire",
+            "Playfully flirted with cats from the other clans at the gathering",
```
```suggestion
            "Playfully flirted with cats from the other Clans at the Gathering",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Is playfully teasing {PRONOUN/m_c/poss} clanmates",
+            "Told a hilarious joke that spread around camp like a wildfire",
+            "Playfully flirted with cats from the other clans at the gathering",
+            "Dazzles {PRONOUN/m_c/poss} clanmates with {PRONOUN/m_c/poss} excellent skills",
```
```suggestion
            "Dazzles {PRONOUN/m_c/poss} Clanmates with {PRONOUN/m_c/poss} excellent skills",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Works to show off {PRONOUN/m_c/poss} skills in every movement {PRONOUN/m_c/subject} {VERB/m_c/make/makes}",
+            "Carefully curates {PRONOUN/m_c/poss} appearance to be the most dazzling in the clan",
+            "Is playfully teasing {PRONOUN/m_c/poss} clanmates",
+            "Told a hilarious joke that spread around camp like a wildfire",
```
```suggestion
            "Told a hilarious joke that spread around camp like wildfire",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Can't stop flexing {PRONOUN/m_c/poss} claws",
+            "Woke up with a flame in {PRONOUN/m_c/poss} belly this morning!",
+            "Is lecturing about something someone said wrong offhandedly",
+            "Taunted a dog stuck in a twoleg's fence",
```
```suggestion
            "Taunted a dog stuck in a Twoleg fence",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Would follow {PRONOUN/m_c/poss} Clanmates to the ends of the earth",
+            "Feels safe within the walls of the camp",
+            "Is promising to always serve StarClan",
+            "Wonders if everyone thinks their own thoughts or if starclan puts their thoughts in their heads",
```
```suggestion
            "Wonders if everyone thinks their own thoughts, or if StarClan puts their thoughts in their heads",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Prays to StarClan for the day to go well",
+            "Is hoping that StarClan protects {PRONOUN/m_c/poss} loved ones...",
+            "Prays to StarClan that next season will be a good one",
+            "Rolls {PRONOUN/m_c/poss} eyes at the kits laughing about spirits not being real",
```
should only happen if there _are_ kits?

Yzzuwyzzuf replied:
yes it should, making a category for it

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Convinced a clanmate to get {PRONOUN/m_c/object} something {PRONOUN/m_c/subject} could easily have gotten {PRONOUN/m_c/self}",
+            "Is always five steps ahead of {PRONOUN/m_c/poss} clanmates",
+            "Knows exactly how to get {PRONOUN/m_c/self} of a mess {PRONOUN/m_c/subject} orchestrated",
+            "Carefully curates {PRONOUN/m_c/poss} appearance to be the most dazzling in the clan",
```
```suggestion
            "Carefully curates {PRONOUN/m_c/poss} appearance to be the most dazzling in the Clan",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+    {
+        "id": "cunning_gen",
+        "thoughts": [
+            "Convinced a clanmate to get {PRONOUN/m_c/object} something {PRONOUN/m_c/subject} could easily have gotten {PRONOUN/m_c/self}",
```
```suggestion
            "Convinced a Clanmate to get {PRONOUN/m_c/object} something {PRONOUN/m_c/subject} could easily have gotten {PRONOUN/m_c/self}",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+        "id": "cunning_gen",
+        "thoughts": [
+            "Convinced a clanmate to get {PRONOUN/m_c/object} something {PRONOUN/m_c/subject} could easily have gotten {PRONOUN/m_c/self}",
+            "Is always five steps ahead of {PRONOUN/m_c/poss} clanmates",
```
```suggestion
            "Is always five steps ahead of {PRONOUN/m_c/poss} Clanmates",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Is always five steps ahead of {PRONOUN/m_c/poss} clanmates",
+            "Knows exactly how to get {PRONOUN/m_c/self} of a mess {PRONOUN/m_c/subject} orchestrated",
+            "Carefully curates {PRONOUN/m_c/poss} appearance to be the most dazzling in the clan",
+            "Got a clanmate to do {PRONOUN/m_c/poss} work with a false promise {PRONOUN/m_c/subject} never intended to fufill, knowing {PRONOUN/m_c/subject} could talk {PRONOUN/m_c/self} out of it easy"
```
```suggestion
            "Got a Clanmate to do {PRONOUN/m_c/poss} work with a false promise {PRONOUN/m_c/subject} never intended to fulfill, knowing {PRONOUN/m_c/subject} could talk {PRONOUN/m_c/self} out of it easy"
```
this one is a little long?

Yzzuwyzzuf replied:
definitely true, will shorten it in a sec

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Makes every patrol into a competition",
+            "Is lecturing about something someone said wrong offhandedly",
+            "Puffs out {PRONOUN/m_c/poss} chest, feeling secure with the space {PRONOUN/m_c/subject} {VERB/m_c/take/takes} up",
+            "Watched a clanmate do a chore they're really good at to see how {PRONOUN/m_c/object} can improve",
```
```suggestion
            "Watched a Clanmate do a chore {PRONOUN/r_c/subject}{VERB/r_c/'re/'s} really good at to see how {PRONOUN/m_c/object} can improve",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Is lecturing about something someone said wrong offhandedly",
+            "Puffs out {PRONOUN/m_c/poss} chest, feeling secure with the space {PRONOUN/m_c/subject} {VERB/m_c/take/takes} up",
+            "Watched a clanmate do a chore they're really good at to see how {PRONOUN/m_c/object} can improve",
+            "Is practicing doing things around camp as fast as possible just in case"
```
```suggestion
            "Is practicing doing things around camp as fast as possible, just in case"
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Challenged a clanmate to a race",
+            "Challenged {PRONOUN/m_c/poss} clanmates to a scavenger hunt",
+            "Challenged {PRONOUN/m_c/poss} clanmates to a scavenger hunt and won",
+            "Challenged {PRONOUN/m_c/poss} clanmates to a scavenger hunt and lost",
```
```suggestion
            "Challenged {PRONOUN/m_c/poss} Clanmates to a scavenger hunt and lost",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Has been a bit of a bad sport about losing to a clanmate",
+            "Challenged a clanmate to a race",
+            "Challenged {PRONOUN/m_c/poss} clanmates to a scavenger hunt",
+            "Challenged {PRONOUN/m_c/poss} clanmates to a scavenger hunt and won",
```
```suggestion
            "Challenged {PRONOUN/m_c/poss} Clanmates to a scavenger hunt and won",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Has been a good sport about losing to a clanmate",
+            "Has been a bit of a bad sport about losing to a clanmate",
+            "Challenged a clanmate to a race",
+            "Challenged {PRONOUN/m_c/poss} clanmates to a scavenger hunt",
```
```suggestion
            "Challenged {PRONOUN/m_c/poss} Clanmates to a scavenger hunt",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Challenged a clanmate to a staring contest and lost",
+            "Has been a good sport about losing to a clanmate",
+            "Has been a bit of a bad sport about losing to a clanmate",
+            "Challenged a clanmate to a race",
```
```suggestion
            "Challenged a Clanmate to a race",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Challenged a clanmate to a staring contest and won",
+            "Challenged a clanmate to a staring contest and lost",
+            "Has been a good sport about losing to a clanmate",
+            "Has been a bit of a bad sport about losing to a clanmate",
```
```suggestion
            "Has been a bit of a bad sport about losing to a Clanmate",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Challenged a clanmate to a staring contest",
+            "Challenged a clanmate to a staring contest and won",
+            "Challenged a clanmate to a staring contest and lost",
+            "Has been a good sport about losing to a clanmate",
```
```suggestion
            "Has been a good sport about losing to a Clanmate",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Is pestering other cats to challenges around camp",
+            "Challenged a clanmate to a staring contest",
+            "Challenged a clanmate to a staring contest and won",
+            "Challenged a clanmate to a staring contest and lost",
```
```suggestion
            "Challenged a Clanmate to a staring contest and lost",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+        "thoughts": [
+            "Is pestering other cats to challenges around camp",
+            "Challenged a clanmate to a staring contest",
+            "Challenged a clanmate to a staring contest and won",
```
```suggestion
            "Challenged a Clanmate to a staring contest and won",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+        "id": "competitive_gen",
+        "thoughts": [
+            "Is pestering other cats to challenges around camp",
+            "Challenged a clanmate to a staring contest",
```
```suggestion
            "Challenged a Clanmate to a staring contest",
```

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Is preoccupied playing an important game of hide and seek",
+            "Is laughing gleefully!",
+            "Gets in an argument over something utterly trivial",
+            "Stuck {PRONOUN/m_c/poss} tongue out at a mouse that escaped {PRONOUN/m_c/poss} pounce",
```
remove this one from here since it doesn't apply to medicine cat( apprentice)s and mediator( apprentice)s?

Yzzuwyzzuf replied:
moved to a better category

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Is helping to reinforce the walls of the elders' den",
+            "Fortifies the camp walls for the fourth time this week",
+            "Feels at peace",
+            "Keeps an eye on some rowdy kits"
```
should only happen if there _are_ kits?

Yzzuwyzzuf replied:
definitely, fixing soon

keplerstelescope commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1424 @@
+            "Is purring quietly",
+            "Has been reaching meditative states while sunbathing",
+            "Calmly takes the lead while some Clanmates argue over this and that",
+            "Is unable to be scared, no matter how much the kits try",
```
should only happen if there _are_ kits?

Yzzuwyzzuf replied:
definitely right about that! fixing soon

keplerstelescope reviewed (commented):
big bundle of edits! mostly spelling/grammar/capitalization fixes

Yzzuwyzzuf committed:
Update resources/dicts/thoughts/alive/kitten.json

committed a suggested wording change

Co-authored-by: keplerstelescope <168871258+keplerstelescope@users.noreply.github.com>

Yzzuwyzzuf committed:
Update resources/dicts/thoughts/alive/kitten.json

committed a suggested wording change

Co-authored-by: keplerstelescope <168871258+keplerstelescope@users.noreply.github.com>

Yzzuwyzzuf committed:
Update resources/dicts/thoughts/alive/kitten.json

committed a suggested change

Co-authored-by: keplerstelescope <168871258+keplerstelescope@users.noreply.github.com>

Yzzuwyzzuf committed:
Update resources/dicts/thoughts/alive/kitten.json

committed a suggested change

Co-authored-by: keplerstelescope <168871258+keplerstelescope@users.noreply.github.com>

Yzzuwyzzuf committed:
Update resources/dicts/thoughts/alive/kitten.json

committed a suggested change

Co-authored-by: keplerstelescope <168871258+keplerstelescope@users.noreply.github.com>

Yzzuwyzzuf committed:
Update resources/dicts/thoughts/alive/general.json

committed a suggested change

Co-authored-by: keplerstelescope <168871258+keplerstelescope@users.noreply.github.com>

Yzzuwyzzuf committed:
Update resources/dicts/thoughts/alive/general.json

committed a suggested change

Co-authored-by: keplerstelescope <168871258+keplerstelescope@users.noreply.github.com>

Yzzuwyzzuf committed:
Update resources/dicts/thoughts/alive/kitten.json

committed a suggested change

Co-authored-by: keplerstelescope <168871258+keplerstelescope@users.noreply.github.com>

Yzzuwyzzuf committed:
Update resources/dicts/thoughts/alive/kitten.json

committed a suggested change

Co-authored-by: keplerstelescope <168871258+keplerstelescope@users.noreply.github.com>

Yzzuwyzzuf committed:
Update resources/dicts/thoughts/alive/kitten.json

Co-authored-by: keplerstelescope <168871258+keplerstelescope@users.noreply.github.com>

Yzzuwyzzuf committed:
Update resources/dicts/thoughts/alive/kitten.json

Co-authored-by: keplerstelescope <168871258+keplerstelescope@users.noreply.github.com>

Yzzuwyzzuf committed:
Update resources/dicts/thoughts/alive/kitten.json

Co-authored-by: keplerstelescope <168871258+keplerstelescope@users.noreply.github.com>

Yzzuwyzzuf committed:
Update resources/dicts/thoughts/alive/kitten.json

Co-authored-by: keplerstelescope <168871258+keplerstelescope@users.noreply.github.com>

Yzzuwyzzuf committed:
Update resources/dicts/thoughts/alive/kitten.json

Co-authored-by: keplerstelescope <168871258+keplerstelescope@users.noreply.github.com>

Yzzuwyzzuf committed:
Apply suggestions from code review

Co-authored-by: keplerstelescope <168871258+keplerstelescope@users.noreply.github.com>

Yzzuwyzzuf committed:
Apply suggestions from code review

Co-authored-by: keplerstelescope <168871258+keplerstelescope@users.noreply.github.com>

Yzzuwyzzuf committed:
Apply suggestions from code review

Co-authored-by: keplerstelescope <168871258+keplerstelescope@users.noreply.github.com>

Yzzuwyzzuf committed:
Update resources/dicts/thoughts/alive/general.json

Co-authored-by: keplerstelescope <168871258+keplerstelescope@users.noreply.github.com>

Yzzuwyzzuf committed:
Update resources/dicts/thoughts/alive/general.json

Co-authored-by: keplerstelescope <168871258+keplerstelescope@users.noreply.github.com>

Yzzuwyzzuf committed:
Update general.json

added tons of helpful spelling, grammar, and capitalization fixes made by keplerstelescope

Yzzuwyzzuf committed:
Update general.json

Yzzuwyzzuf committed:
Update general.json

+1 to gen sibling adult
added parent to their child thoughts +4

Yzzuwyzzuf committed:
Update general.json

fixed a tagging issue with sibling tags

Yzzuwyzzuf committed:
Update general.json

removed gen older adult and the thought that was in it because it was appearing way too often

scribblecrumb commented on resources/dicts/thoughts/alive/apprentice.json:
```diff
@@ -34,7 +34,20 @@
+            "Is relaxing in between training and patrols",
+            "Hopes to have an easy training session",
+            "Is dreading {PRONOUN/m_c/poss} apprentice duties",
+            "Turns out, prey is much harder to hunt then bugs",
```
not the correct phrasing for a thought, could try this instead
```suggestion
            "Discovers that prey is much harder to hunt than bugs",
```

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -4,6 +4,7 @@
         "thoughts": [
             "Is feeling quite lazy",
             "Is spending a considerable amount of time grooming",
+            "Is spending a considerable amount of time grooming",
```
a duplicate thought?

Yzzuwyzzuf replied:
I think I added it twice since cats do spend a lot of time grooming and thought it would make a bit of sense to appear slightly more often but yeah now that i think about it i think ill remove it :)

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+            "Today is just not m_c's day...",
+            "Is regretting not going to sleep sooner...",
+            "Is tidying up camp",
+            "Is tidying up camp",
```
duplicate?

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+    {
+        "id": "gen_not_kit",
+        "thoughts": [
+            "Is thinking on {PRONOUN/m_c/poss} life choices",
```
```suggestion
            "Is thinking about {PRONOUN/m_c/poss} life choices",
```

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+            "Is sharpening {PRONOUN/m_c/poss} claws",
+            "Is hoping {PRONOUN/m_c/subject} find an opportunity to slip back to {PRONOUN/m_c/poss} nest",
+            "Can't wait to take a nap later",
+            "Had a nice chat with a cat from another clan at the border",
```
```suggestion
            "Had a nice chat with a cat from another Clan at the border",
```

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+            "Is hoping {PRONOUN/m_c/subject} find an opportunity to slip back to {PRONOUN/m_c/poss} nest",
+            "Can't wait to take a nap later",
+            "Had a nice chat with a cat from another clan at the border",
+            "Had a nice chat with a loner outside clan territory",
```
```suggestion
            "Had a nice chat with a loner outside Clan territory",
```

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
             "Saw a big scary dog today",
-            "Is looking forward to the next Gathering"
+            "Is drinking water from a nearby stream",
+            "Thought {PRONOUN/m_c/subject} saw a cat but turns out it was a twoleg object that looked like one"
```
```suggestion
            "Thought {PRONOUN/m_c/subject} saw a cat but turns out it was a Twoleg object that looked like one"
```

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+        ]
+    },
+    {
+        "id": "working_cat_warrior",
```
i would suggest separating this into two though categories, one for random dep and one for random leader.  since some of these thoughts are deputy specific, you could end up in a situation where the clan has a leader but not a deputy, resulting in a cat having a thought about a deputy that doesn't exist.

Yzzuwyzzuf replied:
added deputy segment and left the rest for both since no thought mentions leader or could be specific to just leader

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+        ]
+    },
+    {
+        "id": "ambitious_gen",
```
some of these thoughts should be separated into a new thought group that requires the leader as the random cat

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+            "Wonders how {PRONOUN/m_c/subject} would run the Clan better",
+            "Licks fresh-kill blood off {PRONOUN/m_c/poss} lips a little too intently",
+            "Has been glaring at a certain cat in camp with an odd look in {PRONOUN/m_c/poss} eye",
+            "Wonders how easy it'd be to attack the leader from where they lay",
```
needs to be a separate thought group with random leader cat specified and pronoun tagged

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+            "Bravely offers to help the medicine cat pick herbs from a dangerous spot",
+            "Runs out of {PRONOUN/m_c/poss} den into 'battle', only to find that it was just a storm",
+            "Prances around with {PRONOUN/m_c/poss} tail up high, ready for anything",
+            "Promises the kits {PRONOUN/m_c/subject}'ll protect them when the time comes",
```
needs to be in a thought group with random kits specified

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+        "id": "careful_gen",
+        "thoughts": [
+            "Is double-checking {PRONOUN/m_c/poss} nest for burrs",
+            "Is getting a small scratch checked out by the medicine cat",
```
needs to be a separate thought group with random med cat specified

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+            "Is slowly and methodically grooming {PRONOUN/m_c/self}",
+            "Is chiding a younger cat for being so reckless",
+            "Is padding back and forth, across the camp",
+            "Is helping to reinforce the nursery walls",
```
would be nice to see these "building" thoughts in a separate thought group with the camp keeper skills required

Yzzuwyzzuf replied:
segment has been added, would like a check to see if its worded right as I've never added skill specific thoughts before

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+        ]
+    },
+    {
+        "id": "charismatic_gen",
```
separate out the thoughts that mention another warrior/s into their own thought group that requires a random warrior

Yzzuwyzzuf replied:
Would rewording the warriors to just other cats also work or would you prefer the added section even after that?

scribblecrumb replied:
Either would be fine

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+            "Notices that {PRONOUN/m_c/poss} Clanmates have been nervous around {PRONOUN/m_c/object} lately",
+            "Doesn't say thank you when offered a piece of fresh-kill",
+            "Turns away without saying a word when a Clanmate vents to {PRONOUN/m_c/object}",
+            "Thinks the leader should be harsher on codebreakers",
```
move to random leader specific thought group

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+            "Is patiently standing guard outside of camp",
+            "Is winning a staring contest against a Clanmate",
+            "Is listening to a story {PRONOUN/m_c/subject}{VERB/m_c/'ve/'s} heard many, many times",
+            "Is listening to older cats gripe about their day",
```
should probably go to a thought group that has random elder required

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+            "Is waiting to watch the sun set",
+            "Offers to help clanmates with their duties",
+            "Is grooming, grooming, grooming away...",
+            "Smiles at some kit's cheap prank, knowing they'll grow out of it soon",
```
needs to be in a thought group with random kitten specified and pronoun tagged

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+            "Offers a helping paw, even after a night of bad sleep",
+            "Feels {PRONOUN/m_c/poss} kindness might be taken advantage of",
+            "Is listening to the woes of a fellow Clanmate",
+            "Notices an apprentice struggling with a task and offers {PRONOUN/m_c/poss} help",
```
thoughts that mention an apprentice need to go in a thought group that required random apprentice

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+            "Notices an apprentice struggling with a task and offers {PRONOUN/m_c/poss} help",
+            "Is doing some apprentice tasks around camp, to help lighten the load",
+            "Is sharing tongues with friends",
+            "Is keeping the injured cats company",
```
i don't believe we have a way to specify that clan has to have injured cats in it, so probably just remove this one so that nothing incongruent happens

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+            "Is pestering other cats to challenges around camp",
+            "Challenged a Clanmate to a staring contest",
+            "Challenged a Clanmate to a staring contest and won",
+            "Challenged a clanmate to a staring contest and lost",
```
```suggestion
            "Challenged a Clanmate to a staring contest and lost",
```

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+        "thoughts": [
+            "Is puffing {PRONOUN/m_c/poss} chest out",
+            "Is lecturing about something someone said wrong offhandedly",
+            "Thinks that {PRONOUN/m_c/subject} {VERB/m_c/are/is} the smartest cat in the Clan",
```
might be nice to see some of these "best of the clan" thoughts moved into thought groups that require the associated skill.  confident claims feel like they should come from some truth, as opposed to source-less claims an arrogant cat might make.  would additionally be cool to see corresponding thoughts of this nature added for *all* the different skill trees, however that's just a suggestion, not a requirement from me.

Yzzuwyzzuf replied:
will add after confirmation on my other skill related thing

Yzzuwyzzuf replied:
Moving those thoughts to arrogant actually

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+    {
+        "id": "cunning_gen",
+        "thoughts": [
+            "Convinced a clanmate to get {PRONOUN/m_c/object} something {PRONOUN/m_c/subject} easily could've gotten {PRONOUN/m_c/self}",
```
```suggestion
            "Convinced a Clanmate to get {PRONOUN/m_c/object} something {PRONOUN/m_c/subject} easily could've gotten {PRONOUN/m_c/self}",
```

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+        "id": "cunning_gen",
+        "thoughts": [
+            "Convinced a clanmate to get {PRONOUN/m_c/object} something {PRONOUN/m_c/subject} easily could've gotten {PRONOUN/m_c/self}",
+            "Is always five steps ahead of {PRONOUN/m_c/poss} clanmates",
```
```suggestion
            "Is always five steps ahead of {PRONOUN/m_c/poss} Clanmates",
```

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+            "Convinced a clanmate to get {PRONOUN/m_c/object} something {PRONOUN/m_c/subject} easily could've gotten {PRONOUN/m_c/self}",
+            "Is always five steps ahead of {PRONOUN/m_c/poss} clanmates",
+            "Knows exactly how to get {PRONOUN/m_c/self} of a mess {PRONOUN/m_c/subject} orchestrated",
+            "Carefully curates {PRONOUN/m_c/poss} appearance to be the most dazzling in the clan",
```
```suggestion
            "Carefully curates {PRONOUN/m_c/poss} appearance to be the most dazzling in the Clan",
```

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+            "Is always five steps ahead of {PRONOUN/m_c/poss} clanmates",
+            "Knows exactly how to get {PRONOUN/m_c/self} of a mess {PRONOUN/m_c/subject} orchestrated",
+            "Carefully curates {PRONOUN/m_c/poss} appearance to be the most dazzling in the clan",
+            "Got a clanmate to do {PRONOUN/m_c/poss} work never intending to pay {PRONOUN/r_c/object} back for it"
```
```suggestion
            "Got a Clanmate to do {PRONOUN/m_c/poss} work never intending to pay {PRONOUN/r_c/object} back for it"
```

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+            "Is letting a wound go untreated in hopes it gives {PRONOUN/m_c/object} a scar",
+            "Is leaping from boulder to boulder out in the forest",
+            "Is climbing to the top of a very tall tree!",
+            "Taunted a dog stuck behind a twoleg's fence",
```
```suggestion
            "Taunted a dog stuck behind a Twoleg's fence",
```

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+            ]
+    },
+    {
+        "id": "gen_faithful",
```
We don't really consider "faithful" to always mean "loyal to StarClan", rather it means a cat would be faithful to their friends a beliefs, whatever those are.  You could take the StarClan specific thoughts here and move them to a thought group that requires the Star skill, I think that would make more sense with how we characterize faithful cats.

Yzzuwyzzuf replied:
Will remove the starclan thoughts until I properly understand adding skill thoughts

Yzzuwyzzuf replied:
Adding starclan segment now that I understand it better but would appreciate confirmation it looks good

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+        "id": "gen_faithful",
+        "thoughts": [
+            "Rolls {PRONOUN/m_c/poss} eyes at the kits laughing about spirits not being real",
+            "Is scaring the kits with stories of the Dark forest"
```
needs the random status to be kitten

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+        ]   
+    },
+    {
+        "id": "loving_warrior",
```
thoughts that mention another cat of specific status need to be in a thought group with that status required

Yzzuwyzzuf replied:
Changed warriors to clanmates

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+            "Is finding ways to rebel against the instructions {PRONOUN/m_c/subject} were given",
+            "Is doubting why things are done the way they are",
+            "Taunted a dog stuck behind a Twoleg fence",
+            "Started an argument with the leader during a Clan meeting",
```
needs to be in a thought group with random leader required

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+        "thoughts": [
+            "Is planning to wake up early tomorrow",
+            "Was the first to rise this morning",
+            "Reminds an apprentice to follow the code",
```
needs to be in a thought group with random apprentice required

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+        ]
+    },
+    {
+        "id": "righteous_warrior",
```
We don't characterize "righteous" as always meaning "follows StarClan", rather it's a cat who has deeply held beliefs that they strongly believe everyone should hold to.  You could certainly have a righteous cat who thinks their belief in the Dark Forest is unquestionable.  Or a righteous cat who doesn't believe in SC or DF and instead has a strong moral code of their own that they hold to.  You could move some of the SC specific thoughts to a thought group that requires the Star skill for now.

Yzzuwyzzuf replied:
Same as what I said for faithful

Yzzuwyzzuf replied:
Added starclan segment and re-added some thoughts there

scribblecrumb reviewed (changes requested):
Additionally, just in general, you should make sure that if a thought mentions another cat status (warrior, leader, dep, ect.) then the random cat *has* to require that status.  there's a plethora of situations in which there may only be one warrior in a clan, or no leader or deputy, or no med cat, ect.  

Also, it would be nice to see less thoughts that just say "Clanmate" rather than actually including a second cat's name and pronouns.  If you have time and energy, I'd love if you could go through your additions in this PR and change some of those.  However, that's not a requirement for this PR being accepted, just a suggestion.

I haven't gotten through this whole PR yet, but this is what I managed to do this morning lol.

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+            "Is bragging about how great a hunter {PRONOUN/m_c/subject} are",
+            "Announced that {PRONOUN/m_c/subject} {VERB/m_c/are/is} heading to the dirtplace",
+            "Taunted a dog stuck behind a Twoleg fence",
+            "Is eating, chewing very loudly",
```
```suggestion
            "Is eating and chewing very loudly",
```
felt the phrasing was a little awkward

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+        ]
+    },
+	{
+        "id": "sincere_mediator_questioning",
```
might be nice to see a "duplicate" group of this that's aimed at mediator and speaker skill cats rather than sincere trait cats!

Yzzuwyzzuf replied:
Will fix after confirmation on other skill related segments

Yzzuwyzzuf replied:
Added mediator segment

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+            "Volunteered to guard the dirtplace...?",
+            "Is telling stories that only make sense to {PRONOUN/m_c/object}",
+            "Doesn't feel understood by {PRONOUN/m_c/poss} Clanmates",
+            "Wandered off alone.",
```
```suggestion
            "Wandered off alone",
```

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+            "Is making odd noises",
+            "Is tearing up {PRONOUN/m_c/poss} food",
+            "Is gnawing on a stick",
+            "Collects the bones of prey",
```
```suggestion
            "Is collecting the bones of prey",
```

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+            "Is drawing symbols in the dirt with {PRONOUN/m_c/poss} claws",
+            "Is sitting quietly on the far side of camp",
+            "Paces back and forth",
+            "Wonders if everyone thinks their own thoughts or if starclan puts thoughts in their heads",
```
```suggestion
            "Wonders if everyone thinks their own thoughts or if StarClan puts thoughts in their heads",
```

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+            "Wonders if everyone thinks their own thoughts or if starclan puts thoughts in their heads",
+            "Is zoning out",
+            "Found another secret exit out of camp... somehow",
+            "Is asking about these 'cameras' no other cat seems to notice",
```
mmmmm a little too meta for the official version, i think this one should be removed

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+        "thoughts": [
+            "Is offering helpful advice to a sad Clanmate",
+            "Prepared the perfect gift for a friend",
+            "Noticed a Clanmate was tired and offersed to take on their chores",
```
```suggestion
            "Noticed a Clanmate was tired and offered to take on their chores",
```

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
-        ]
+            "Is urging r_c to rest",
+            "Brought r_c some prey",
+            "Wonders if {PRONOUN/m_c/subject}'ll still be accepted into StarClan after taking r_c as a mate",
```
We currently don't include the "med cats can't have mates" part of canon within official ClanGen.  The dev team is still discussing how best to integrate it within the game, if we do at all.  So, for now, these three thoughts should be removed.

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -51,87 +49,1497 @@
+        "thoughts": [
+            "Play fought with r_c just like old times",
+            "Reminisced with r_c on when they were kits",
+            "Took r_c's fresh_kill {PRONOUN/r_c/subject} were saving because {PRONOUN/r_c/subject} were saving it", 
```
```suggestion
            "Took r_c's fresh_kill that {PRONOUN/r_c/subject} were saving", 
```
Gets a little confusing, I think it's clearer without the last bit

Yzzuwyzzuf replied:
I actually worded it that way because its meant to be them intentinally taking the prey specifically because they knew their sibling was saving it. Will reword it to be more clearer

Yzzuwyzzuf replied:
Wording hopefully better now

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -503,6 +1948,24 @@
+    {
+        "id": "gen_to_dead_parent",
+        "thoughts":[
+            "Heard r_c humming a tune in {PRONOUN/m_c/poss} dream {PRONOUN/m_c/subject} remembered {PRONOUN/r_c/object} humming to {PRONOUN/m_c/object} in the nursery",
```
```suggestion
            "Heard r_c humming a tune in {PRONOUN/m_c/poss} dream that {PRONOUN/m_c/subject} remembered r_c humming in the nursery",
```
Just cutting down on some of the pronoun usage, flip flopping between two pronoun sets without interjecting some names makes it confusing, esp if both cats use the same pronouns

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -1796,6 +3323,153 @@
+    {
+        "id": "gen_newleaf",
+        "thoughts": [
+            "Enjoying the warmth of Newleaf",
```
```suggestion
            "Enjoying the warmth of newleaf",
```

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -1796,6 +3323,153 @@
+        "id": "gen_newleaf",
+        "thoughts": [
+            "Enjoying the warmth of Newleaf",
+            "Is relieved Leaf-bare is finally over",
```
```suggestion
            "Is relieved leaf-bare is finally over",
```

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -1796,6 +3323,153 @@
+        "thoughts": [
+            "Enjoying the warmth of Newleaf",
+            "Is relieved Leaf-bare is finally over",
+            "Is soaking in the Newleaf warmth",
```
```suggestion
            "Is soaking in the newleaf warmth",
```

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -1796,6 +3323,153 @@
+    {
+        "id": "gen_greenleaf",
+        "thoughts": [
+            "Enjoying the warmth of Greenleaf",
```
```suggestion
            "Enjoying the warmth of greenleaf",
```

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -1796,6 +3323,153 @@
+    {
+        "id": "gen_leaffall_not_kit",
+        "thoughts": [
+            "Is worried about the lack of prey Leaf-bare will bring",
```
```suggestion
            "Is worried about the lack of prey leaf-bare will bring",
```

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -1796,6 +3323,153 @@
+        "thoughts": [
+            "Is worried about the lack of prey Leaf-bare will bring",
+            "Is clearing the clearing of fallen leaves",
+            "Caught extra prey for Leaf-bare",
```
```suggestion
            "Caught extra prey for leaf-bare",
```

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -1796,6 +3323,153 @@
+            "Is worried about the lack of prey Leaf-bare will bring",
+            "Is clearing the clearing of fallen leaves",
+            "Caught extra prey for Leaf-bare",
+            "Is not looking forward to the sparse hunting in Leaf-bare"
```
```suggestion
            "Is not looking forward to the sparse hunting in leaf-bare"
```

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -1796,6 +3323,153 @@
+        "thoughts": [
+            "Sneezed when a snowflake landed on {PRONOUN/m_c/poss} nose",
+            "Took a nap in the clearing and woke up under a blanket of snow",
+            "Can't wait for Newleaf",
```
```suggestion
            "Can't wait for newleaf",
```

Yzzuwyzzuf committed:
Update resources/dicts/thoughts/alive/apprentice.json

Added suggested wording change

Co-authored-by: scribblecrumb <115410010+scribblecrumb@users.noreply.github.com>

scribblecrumb commented on resources/dicts/thoughts/alive/kitten.json:
```diff
@@ -14,6 +14,24 @@
+            "Is sleeping where r_c used to sleep",
+            "Went to ask r_c to play only to remember {PRONOUN/r_c/subject}{VERB/r_c/'re/'s} gone...",
+            "Is crying for r_c",
+            "Dosen't understand why r_c won't wake up"
```
```suggestion
            "Doesn't understand why r_c won't wake up"
```

scribblecrumb commented on resources/dicts/thoughts/alive/kitten.json:
```diff
@@ -30,6 +48,40 @@
         ]
     },
+    {
+        "id": "kit_med_parent",
```
need to remove the thoughts here about med cat parents being codebreakers

Yzzuwyzzuf replied:
its been reworded to boring medicine cat parents instead :)

scribblecrumb commented on resources/dicts/thoughts/alive/kitten.json:
```diff
@@ -68,7 +120,40 @@
             "Wants to have fun",
-            "Is playing pretend"
+            "Is playing pretend",
+            "Is going through an angsty phase much to {PRONOUN/m_c/poss} clanmates chagrin",
```
```suggestion
            "Is going through an angsty phase much to {PRONOUN/m_c/poss} Clanmates chagrin",
```

scribblecrumb commented on resources/dicts/thoughts/alive/kitten.json:
```diff
@@ -68,7 +120,40 @@
+            "Is telling on r_c for climbing a bit too high",
+            "Is telling on r_c for doing something bad",
+            "Is inviting r_c to play",
+            "Dosen't want to play with r_c",
```
```suggestion
            "Doesn't want to play with r_c",
```

scribblecrumb commented on resources/dicts/thoughts/alive/kitten.json:
```diff
@@ -177,6 +290,34 @@
+            "Thinks {PRONOUN/m_c/subject} can outsmart a fox",
+            "Wants to guard the nursery",
+            "Is sure {PRONOUN/m_c/subject} could fight a badger",
+            "Is sneaking into the leader's nest",
```
need to move this to a random leader specific thought group

scribblecrumb commented on resources/dicts/thoughts/alive/kitten.json:
```diff
@@ -177,6 +290,34 @@
+            "Wants to guard the nursery",
+            "Is sure {PRONOUN/m_c/subject} could fight a badger",
+            "Is sneaking into the leader's nest",
+            "Picks a fight with an apprentice so they can show their skills",
```
need to move this to a random app specific thought group

Yzzuwyzzuf replied:
it had been moved now :)

Yzzuwyzzuf committed:
Update general.json

Removed a duplicate thought

scribblecrumb commented on resources/dicts/thoughts/alive/kitten.json:
```diff
@@ -440,9 +611,11 @@
-            "Is practicing bird calls"
+            "Is practicing bird calls",
+            "Is screaming because they can",
+            "Asked an elder for a story but keeps interrupting them"
```
needs to be in a random elder specific thought group

scribblecrumb reviewed (changes requested):
aaaand that's all the reviewing done!

Yzzuwyzzuf committed:
Update kitten.json

added all suggested changes

Yzzuwyzzuf committed:
Update general.json

committed all suggested hanges

Yzzuwyzzuf committed:
Update general.json

fixed small error

Yzzuwyzzuf committed:
Update general.json

fixed error

Yzzuwyzzuf committed:
Update general.json

added suggested changes

Yzzuwyzzuf committed:
Update general.json

fixed one last thought I missed

Yzzuwyzzuf reviewed (commented):
just need these reviewed rq

Yzzuwyzzuf committed:
Update general.json

added skill segments

Yzzuwyzzuf reviewed (commented):
Update to last review now that I got more info

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -1391,6 +1475,71 @@
+	],
+	"main_skill_constraint": [
+		"STAR,1"
+	],
```
```suggestion
	]
```
stray comma

Yzzuwyzzuf replied:
fixed

scribblecrumb commented on resources/dicts/thoughts/alive/kitten.json:
```diff
@@ -52,7 +52,7 @@
-            "Is being teased for having codebreakers as parents",
+            "Is being teased for having boring medicine cats as parents",
             "Wonders if {PRONOUN/m_c/subject}'ll receive prophecies like r_c",
             "Wonders if StarClan hates {PRONOUN/m_c/object} for being a medicine cat's kit",
```
this one also needs to be removed!

Yzzuwyzzuf replied:
dont know how i completely missed that! fixing now!

Yzzuwyzzuf replied:
Thought you meant there was an extra iteration of the codebreaker one. Removed and will consider a way to re-add to bullying kit in the future

scribblecrumb commented on resources/dicts/thoughts/alive/kitten.json:
```diff
@@ -318,6 +317,33 @@
+        "main_trait_constraint": [
+            "daring"
+        ],
+        "random_status_constraint" [
```
```suggestion
        "random_status_constraint": [
```
missing colon

scribblecrumb commented on resources/dicts/thoughts/alive/kitten.json:
```diff
@@ -318,6 +317,33 @@
+    {
+        "id": "daring_kit",
+        "thoughts": [
+            "Picks a fight with an apprentice so they can show their skills"
```
missing pronoun tags

Yzzuwyzzuf replied:
Fixed

scribblecrumb commented on resources/dicts/thoughts/alive/kitten.json:
```diff
@@ -624,6 +647,18 @@
+    {
+        "id": "noisy_kit",
+        "thoughts": [
+            "Asked an elder for a story but keeps interrupting them"
```
i think it would be nice to actually use r_c's name and pronouns here

Yzzuwyzzuf replied:
Fixed

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -1391,6 +1475,71 @@
+	]
+    },
+	{
+	"id: "starclan_gen",
```
```suggestion
	"id": "starclan_gen",
```
missing quotation

Yzzuwyzzuf replied:
fixed

scribblecrumb commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -1391,6 +1475,71 @@
+	],
+	},
+	{
+	"id: "starclan2_to_kit",
```
```suggestion
	"id": "starclan2_to_kit",
```
missing quotation

Yzzuwyzzuf replied:
fixed

scribblecrumb reviewed (changes requested):
Just a few more bits

Yzzuwyzzuf committed:
Update resources/dicts/thoughts/alive/general.json

Co-authored-by: scribblecrumb <115410010+scribblecrumb@users.noreply.github.com>

Yzzuwyzzuf committed:
Update resources/dicts/thoughts/alive/general.json

Co-authored-by: scribblecrumb <115410010+scribblecrumb@users.noreply.github.com>

Yzzuwyzzuf committed:
Update resources/dicts/thoughts/alive/general.json

Co-authored-by: scribblecrumb <115410010+scribblecrumb@users.noreply.github.com>

Yzzuwyzzuf committed:
Update resources/dicts/thoughts/alive/kitten.json

Co-authored-by: scribblecrumb <115410010+scribblecrumb@users.noreply.github.com>

Yzzuwyzzuf committed:
Update kitten.json

suggested corrections made

Yzzuwyzzuf committed:
Update kitten.json

completely missed a thought needed to be removed

Yzzuwyzzuf committed:
Update kitten.json

pronoun fix

scribblecrumb reviewed (changes requested):
Looking great!  Just get the conflicted file handled and then we can merge this.  json file conflicts can be a pain in the ass, so if it's being particularly messy then you could try just looking back at the recently merged PRs for the couple that made changes to the thoughts file, then manually transfer over whatever they added that you're missing in your version.  Let me know if you have questions on this.

Yzzuwyzzuf committed:
Update apprentice.json

Fixed the conflict

Yzzuwyzzuf committed:
Update apprentice.json

Tried fixing a conflict

scribblecrumb committed:
Merge branch 'development' into Fuzzys-expanded-text-mod

Yzzuwyzzuf committed:
Update kitten.json

deleted 2 extra commas

Yzzuwyzzuf committed:
Update general.json

deleted extra comma
